BLE Indoor Positioning/src/main/java/com/nexenio/bleindoorpositioning/IndoorPositioning.java

```diff
@@ -33,6 +33,7 @@
 
     public static final double MAXIMUM_MOVEMENT_SPEED_NOT_SET = -1;
     private double maximumMovementSpeed = MAXIMUM_MOVEMENT_SPEED_NOT_SET;
+    private int ROOT_MEAN_SQUARE_THRESHOLD = ROOT_MEAN_SQUARE_THRESHOLD_STRICT;
```

Comment:
As the field is not a constant anymore, it should be renamed to `rootMeanSquareThreshold `